src/controllers/queryRunner.ts

```diff
@@ -428,20 +431,25 @@ export default class QueryRunner {
                 return ((a.fromCell < b.fromCell) ? -1 : (a.fromCell > b.fromCell) ? 1 : 0);
             });
 
-            // start copy paste from left-most column
-            const firstColumn = rowSelections[0].fromCell;
-
             for (let i = 0; i < rowSelections.length; i++) {
                 let rowSelection = rowSelections[i];
+
+                // Add tabs starting from the first column of the selection
                 for (let j = firstColumn; j < rowSelection.fromCell; j++) {
                     copyString += ' \t';
```

Comment:
Do you have any idea why we're adding an extra space here? Does that show up in the copied text after your fix? It doesn't seem like that should be included

Reply:
I'm not sure why it was there, good catch!